.github/RUNNING_A_SERVER.md

```diff
@@ -89,6 +89,56 @@ Web delivery of game resources makes it quicker for players to join and reduces
     * If you keep up to date with /tg/ you could reuse /tg/'s rsc cdn at http://tgstation13.download/byond/tgstation.zip. Otherwise you can use cdn services like CDN77 or cloudflare (requires adding a page rule to enable caching of the zip), or roll your own cdn using route 53 and vps providers.
 	* Regardless even offloading the rsc to a website without a CDN will be a massive improvement over the in game system for transferring files.
 
+### All In One Amazon Web Services Hosting and Content delivery network.
+**Important Note**
+It is very Importat to note that since AWS is all highly integrated its "easier" than some solutions. However the Price to Performance Ratio is terrible.
```

Comment:
I don't know if this is strong enough of a warning.

At tg's current bandwidth usage it would cost us something like 6000 usd or something crazy.

maybe mention:
At the scale of a large server like /tg/station, the costs would exceed 1000USD a month, so please be aware of that while planning.

Reply:
yeah not sure what your bandwidth usage is, but I have reviewed the different classes of of processors and dedicated/vs shared cpu time and yeah the costs escalate so fast that buying your own hardware becomes feasible. i think my current operating costs are about $450 per anum. Peek traffic of 20 players with 4 or 5 regulars.

Reply:
I'll try and think of some additional cost warnings or formatting in the morning. brain fried

Reply:
Last up to date info from mso was about 6476 GB in a 30 day period

Reply:
https://calculator.s3.amazonaws.com/index.html#r=IAD&key=files/calc-2bd21e222dbb3a56fcb229a9abb82eea5b956552&v=ver20210420bN

Well, it may only be half baked data since i do not know the ins and outs of TG's set up but its not /that/ bad but that is assuming 1 server instance  handling the entire load of your production servers. although the 3 grand upfront for preferred pricing on the instance is rather nuts

Reply:
yeah but we're paying like 200-300usd a month for two dedicated machines,EU/US, intel 9900k's with like 32G of ram each and several terabytes of nvme ssd storage and 10 terabytes worth of bandwidth, so AWS just isn't even remotely competitive

@MrStonedOne can probably share more.

Anyway your updates look good I see no raisin not to merge.

Reply:
6-7tb on the us server, a little under 1tb on the eu server, and another tb on the cloudflare cdn is about ~8-9tb of monthly usage at /tg/.

we pay about 300 usd for two machines with a tb of raid 1 storage and 1 with 64gb of ram and another with 128gb of ram. each supports 10tb of bw.

I'm not sure we should be trapping new communities into infrastructure that will eventually break their bank, but if we were to do something like this, my suggestion would be to suggest **aws lightsail** over aws ec2. There is enough bandwidth included in the price, the integrations are bare enough to keep you from getting too locked in, and the moment they need faster servers, they can move to another vps provider, ideally one that also does high clock speed servers like ovh or hetzner

Also, any cdn system needs multi-region, the whole point of the cdn is that the clients download it from local servers so that the download is faster since byond uses a poorly scaled chunked sending system that waits for replies to send the next chunk, so lower ping means higher speed. You would want to put aws cloudfront or (better) Cloudflare in front of the rsc.

And finally, this will all become moot in less then a year as we are slowly working on a ansible deployment setup for /tg/ that could be used to deploy things to any vps or dedicated server provider as long as you slap cloudflare in front of it.

Reply:
Yeah the AWS rabbit hole is a dark and a deep one. I largely went down it because of a significant lack of documentation on any alternatives.

I am interested in this new package that is going to be rolled out. Hope there is documentation because after making a cloudfront account i was completely lost